collections/_article/godot-xr-update-aug-2023.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Godot XR update - August 2023"
+excerpt: "Updates on various things XR in Godot, a new version of the OpenXR plugin, a new version of the tools library and an update on Godot 4 support."
+categories: ["progress-report"]
+author: Bastiaan Olij
+image: /storage/blog/covers/godot-xr-update-aug-2023.webp
+date: 2023-08-27 11:00:00
+---
+
+With the last grant provided by Meta Reality Labs having ended in December 2022 the core team's focus on XR has been limited.
+
+There is still plenty happening on the XR front to warrant another update.
+
+We again thank Meta for the support past and present that they have been able to provide.
```

Comment:
I would shift the focus away from Meta in these first couple of paragraphs. There is no reason to thank them for a grant that didn't fund the described work. Similarly, we want to highlight both that we need an XR grant to shift the focus back to XR, and that the grant can come from anyone (It would be great if it came from Meta as we have worked well with them in the past, but other companies are free to step up as well).

I would go with something a little bit subtler
```suggestion
XR work has slowed down this year due to the XR-specific line of funding running out. However, there is still plenty happening on the XR front to warrant another update.
```

Reply:
I didn't want to come across too harsh, but yeah us currently not having any funding for XR has had a measurable impact which is why I wanted to highlight that. I like your rewording